server/test/swagger_2/swagger_2.test.js

```diff
@@ -0,0 +1,76 @@
+const app = require('../../app');
+const supertest = require('supertest');
+const request = supertest(app);
+
+const jsonResponse1 = require('./data/output/jsonResponse1.json')
+const jsonResponse2 = require('./data/output/jsonResponse2.json')
+const { yamlResponse1, yamlResponse2 } = require('./data/output/yamlResponses')
+
+// jsonResponse.replace(/\\"/g, '"');
+jest.setTimeout(30000);
+
+describe('POST /api/specification/swagger_2/openapi_3', () => {
```

Comment:
Can we add a test to check the file download feature?

Reply:
I've added 2 tests in the end that check for the content-disposition header. This header indicates the attachment and the file that will be downloaded.